services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/stage2/review_activities.jsx

```diff
@@ -74,10 +82,30 @@ export default class ReviewActivities extends React.Component {
 
     this.state = {
       snackbarVisible: false,
-      erroredActivityIds: []
+      erroredActivityIds: [],
+      previouslyAssignedActivityData: null
+    }
+  }
+
+  componentDidUpdate() {
```

Comment:
Are we calling this here because `activities` isn't populated when the page loads? Otherwise, I'd just call this in the constructor.